block-filters/separator/index.php

```diff
@@ -39,11 +39,11 @@ function action_enqueue_separator_assets() {
 /**
  * Add a block separator if needed.
  *
- * @param string    $block_content The current block content.
- * @param \WP_Block $block The parse block object.
+ * @param string $block_content The current block content.
+ * @param array  $block The parsed block object.
```

Comment:
Wondering why the actual type didn't work here?

Reply:
It was incorrect! https://developer.wordpress.org/reference/hooks/render_block/